spec/interfaces.md

```diff
@@ -81,192 +100,200 @@ If a generic interface is declared in multiple parts (§15.2.3), each partial de
 The occurrence of variance annotations in the type parameter list of a type restricts the places where types can occur within the type declaration.
 
 A type T is ***output-unsafe*** if one of the following holds:
-
--   T is a contravariant type parameter
-
--   T is an array type with an output-unsafe element type
-
--   T is an interface or delegate type S&lt;A~1~,… A~K~&gt; constructed from a generic type S&lt;X~1~, … X~K~&gt; where for at least one A~i~ one of the following holds:
-
-<!-- -->
-
--   X~i~ is covariant or invariant and A~i~ is output-unsafe.
-
--   X~i~ is contravariant or invariant and A~i~ is input-unsafe.
-
+- `T` is a contravariant type parameter
+- `T` is an array type with an output-unsafe element type
+- `T` is an interface or delegate type S<A<sub>1</sub>,… A<sub>K</sub> constructed from a generic type S<X<sub>1</sub>, … X<sub>K</sub>> where for at least one A<sub>i</sub> one of the following holds:
```

Comment:
This is a special case where markdown doesn't preserve the fidelity in the original word source. We've discussed, and we prefer keeping the code formatting instead of the subscript:

```suggestion
- `T` is an interface or delegate type `Si,… Ak` constructed from a generic type `S<Xi, … Xk>` where for at least one `Ai` one of the following holds:
```